### DISCUSSION TITLE
Are you still working on Fooocus?

### DISCUSSION THREAD

**Discussion by Azur231** [mashb1t/Fooocus#89] [General]
Hey, just a quick question ... since lllyasviel has practically given up on Fooocus or considers it finished, are you still working on maintaining the ‘original version’? 

Because there are many who really enjoy using Fooocus - but find RuinedFooocus and other forks too cluttered -- and would like to see it at least somehow kept up to date or given support for models like Flux or SD 3. 

I think some people would be happy to support you financially via Buy Me A Coffee.

**Comment by mashb1t** [comment]
Short answer: No.
Long answer: see https://github.com/lllyasviel/Fooocus/discussions/3504

I'd suggest to use [SimpleSDXL](https://github.com/metercai/SimpleSDXL), [Forge](https://github.com/lllyasviel/stable-diffusion-webui-forge) or [ComfyUI](https://github.com/comfyanonymous/ComfyUI) instead.
